### DISCUSSION TITLE
[bug] text-generation-inference backend for VLMs does not see images

### DISCUSSION THREAD

**Discussion by lucyknada** [open-webui/open-webui#7760] [General]
issue: sending images to tgi returns nonsense, probably because the image is not sent in a way it expects it to?

![image](https://github.com/user-attachments/assets/dda71a48-44ed-4282-904c-6361a801095d)

ref: https://huggingface.co/docs/text-generation-inference/main/en/basic_tutorials/visual_language_models#hugging-face-hub-python-library

comparing that to the payload openwebui sends via devtools:

![image](https://github.com/user-attachments/assets/dfc65d4c-af42-4066-9995-5aa2d7e18380)

it seems they might expect the markdown embed? `![](base64-here)` or are they unable to use images mid-context and force you to send "inputs" key?

just guessing / mindstorming what the issue could be, thanks!

tracking: https://github.com/huggingface/text-generation-inference/issues/2820

**Comment by tjbck** [comment]
Definitely the correct OpenAI format here, keep us updated!

**Reply by lucyknada** [reply]
thanks! opening an issue with tgi then
